From 2c7ba8f9439804bd3933a8a9ae8f48bb74f388bb Mon Sep 17 00:00:00 2001
From: Shunqian Zheng <zhengsq@rock-chips.com>
Date: Thu, 28 Jun 2018 15:09:25 +0800
Subject: [PATCH] ARM: config: enable CIF and ADV7181D for
 rockchip_linux_defconfig

This enables CONFIG_VIDEO_ROCKCHIP_CIF and CONFIG_VIDEO_ADV7181D
for rockchip_linux_defconfig.

Change-Id: I657e757d80a8e2a581631eedd2b3a19254658883
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index f1cec6866df3..2e3d730734a7 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -295,12 +295,14 @@ CONFIG_USB_VIDEO_CLASS=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_SOC_CAMERA=y
 CONFIG_VIDEO_RK_CIF_ISP10=y
+CONFIG_VIDEO_ROCKCHIP_CIF=y
 CONFIG_VIDEO_ROCKCHIP_ISP1=y
 CONFIG_V4L_MEM2MEM_DRIVERS=y
 CONFIG_VIDEO_ROCKCHIP_RGA=y
 CONFIG_VIDEO_ROCKCHIP_VPU=y
 CONFIG_V4L_TEST_DRIVERS=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
+CONFIG_VIDEO_ADV7181D=y
 CONFIG_VIDEO_TC35874X=y
 CONFIG_VIDEO_IMX219=y
 CONFIG_VIDEO_OV5647=y
-- 
2.35.3

